controls/postgres_spec.rb

```diff
@@ -129,10 +63,8 @@
   impact 1.0
   title 'Run one postgresql instance per operating system'
   desc 'Only one postgresql database instance must be running on an operating system instance (both physical HW or virtualized).'
-  pg_command = 'postgres'
-  pg_command = 'postmaster' if os.redhat? && os.release.include?('6.')
-  describe processes(pg_command) do
-    its('entries.length') { should eq 1 }
+  describe command('ps aux | awk /\'bin\/postmaster\'/ | wc -l') do
```

Comment:
Does this mean that the process is called `postmaster` on all OS'es now? Have you had the possibility to test it?

Reply:
Good point, not realy sure. I will look at this.